plugins/org.eclipse.elk.alg.topdownpacking/src/org/eclipse/elk/alg/topdownpacking/Topdownpacking.melk

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0  
+ *******************************************************************************/
+package org.eclipse.elk.alg.topdownpacking
+
+import org.eclipse.elk.alg.topdownpacking.TopdownpackingLayoutProvider
+import org.eclipse.elk.core.math.ElkPadding
+import org.eclipse.elk.core.options.TopdownNodeTypes
+
+bundle {
+    metadataClass options.TopdownpackingMetadataProvider
+    idPrefix org.eclipse.elk.topdownpacking
+}
+
+algorithm topdownpacking(TopdownpackingLayoutProvider) {
+    label "Topdownpacking"
+    description "An algorithm for placing boxes of fixed sizes. Requires its {@link CoreOptions.TOPDOWN_NODE_TYPE} 
```

Comment:
It seems weird that is requires a specific option to work at all. Is this option the default value?

Reply:
This is a little intertwined, technically this algorithm is necessary for parallel nodes to work. Therefore it should only be used on parallel nodes. Or single hierarchy layouts could also use it, but that doesn't seem very useful